components/Home/Main/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```

Comment:
There is already a component with name Main in documents.tsx file. Can you rename this to MainContent?